mesonbuild/mesonlib.py

```diff
@@ -470,6 +470,17 @@ def replace_if_different(dst, dst_tmp):
     else:
         os.unlink(dst_tmp)
 
+
+def listify(*args):
+    '''
+    Returns a list with all args embedded in a list if they are not of type list.
+    This function preserve order.
```

Comment:
Grammar mistake. Replace with:

This function preserves order.
or
This function will preserve order.